content/zh-cn/blog/falco-0-28-0.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Falco 0.28.0 a.k.a. Falco 2021.04
+date: 2021-04-09
+author: Leonardo Di Donato
+slug: falco-0-28-0
+---
+
+今天我们宣布春季发布Falco 0.28.0
+
+这是Falco在2021年的第二次发布！
+
+您可以在此处查看一组更改：
+
+- 0.28.0 https://github.com/falcosecurity/falco/releases/tag/0.28.0
+
+像往常一样，如果您只想试用稳定的 Falco 0.28.0，您可以按照文档中概述的过程安装其软件包：
+
+- CentOS/Amazon Linux https://falco.org/docs/getting-started/installation/#centos-rhel
+- Debian/Ubuntu https://falco.org/docs/getting-started/installation/#debian
+- openSUSE https://falco.org/docs/getting-started/installation/#suse
+- Linux binary package https://falco.org/docs/getting-started/installation/#linux-binary
+
+您更喜欢使用容器镜像吗？ 一点问题都没有！
+
+您可以在文档中阅读有关使用 Docker 运行 Falco 的更多信息。docs https://falco.org/docs/getting-started/running/#docker.
+
+请注意，从此版本开始，感谢我们的 Falco Infra 维护者之一 Jonah，您还可以在 AWS ECR 库上找到 falco-no-driver 容器镜像https://gallery.ecr.aws/falcosecurity/falco-no-driver。 falco-driver-loader 容器镜像也是如此https://gallery.ecr.aws/falcosecurity/falco-driver-loader。 这是在其他注册中心上发布 Falco 容器镜像的一部分，这些镜像是在制作 Falco 0.27.0 时开始的。
+
+
+## 新鲜事
+
+现在让我们回顾一下 Falco 0.28.0 带来的一些新东西。
+
+如需完整列表，请访问更新日志。https://github.com/falcosecurity/falco/releases/tag/0.28.0
+
+### 重大变化
+
+在我们深入研究之前，重要的是要注意此版本引入了一些重大更改。
```

Comment:
Link is missing: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter
Formatting is missing.